tests/strategies/test_deepspeed_strategy.py

```diff
@@ -1077,3 +1077,23 @@ def test_deepspeed_with_meta_device(tmpdir):
     )
     trainer.fit(model)
     assert model.layer.weight.device.type == "cpu"
+
+
+@RunIf(min_gpus=2, deepspeed=True, standalone=True)
+def test_deepspeed_multi_save_same_filepath(tmpdir):
```

Comment:
can we add a description to the test? especially since this test is to prevent regression

Reply:
done!